torch/ao/quantization/quantizer/xnnpack_quantizer.py

```diff
@@ -417,7 +471,10 @@ def _annotate_conv2d_bn_relu(
             _mark_nodes_as_annotated(nodes_to_mark_annotated)
 
     def _annotate_conv2d_relu(
-        self, gm: torch.fx.GraphModule, quantization_config: QuantizationConfig
+        self,
+        gm: torch.fx.GraphModule,
+        quantization_config: QuantizationConfig,
+        filter_fn: Optional[Callable[[Node], bool]] = None,
     ) -> None:
         fused_partitions = find_sequential_partitions(
             gm, [torch.nn.Conv2d, torch.nn.ReLU]
```

Comment:
What if the user has
```
self.my_conv = torch.nn.Conv2d(...)
self.my_relu = torch.nn.ReLU(...)
```
and they do `set_module_name("my_conv")` and `set_module_name("my_relu")`? I guess we ignore these settings? I guess this may be a bit complicated to support

Reply:
it's not used in this way, user should set_module_name("module_name_for_parent_of_conv_and_relu")

if you are talking about "quantize all conv-relu patterns in my model" I think that is another api, right now it is set_operator_type("conv_relu"), but the quantizer can expose this in a different way as well